rhobs/alerting/data_plane/prometheus.gitops_sync_alerts.yaml

```
@@ -0,0 +1,52 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-sync-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: gitops_sync_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsApplicationSyncErrors
+      expr: |
+        (
+          sum(avg_over_time(argocd_app_info{sync_status="Synced", namespace="gitops-service-argocd"}[5m]))
+          or
+          sum(argocd_app_info{namespace="gitops-service-argocd"}) * 0
+        )
+        /
+        (
+          sum(avg_over_time(argocd_app_info{namespace="gitops-service-argocd"}[5m]))
+          or
+          sum(argocd_app_info{namespace="gitops-service-argocd"}) * 0
+        ) < 0.95
+      for: 5m
+      labels:
+        severity: warning
+        slo: true
+      annotations:
+        summary: >-
+          Less than 95% of GitOps applications are in synced state: {{ $value }}
```

This can make it nicer if you prefer:
```suggestion
          Less than 95% of GitOps applications are in synced state: {{ $value | humanizePercentage }}
```

---

Yes, I prefer this  👍 , will make the change.